projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -121,7 +121,32 @@ export class AppModule {}
 
 Registering states with `StoreModule.forRoot()` ensures that the states are defined upon application startup. In general, you register root states that always need to be available to all areas of your application immediately.
 
-## Register feature state
+### Using the Standalone API
+
+Registering the root store and state can also be done using the standalone APIs if you are bootstrapping an Angular application using standalone features.
+
+<code-example header="main.ts">
+import { bootstrapApplication } from '@angular/platform-browser';
+import { provideStore, provideState } from '@ngrx/store';
+
+import { AppComponent } from './app.component';
+import { scoreboardReducer } from './reducers/scoreboard.reducer';
+
+bootstrapApplication(AppComponent, {
+  providers: [
+    provideStore(),
+    provideState({ game: scoreboardReducer })
+  ],
+});
+</code-example>
+
+<div class="alert is-important">
+
+**Note:** Although you can register reducers in the `provideStore()` function, we recommened keeping `provideStore()` empty and using the `provideState()` function to register feature states in the root `providers` array.
```

Comment:
What do you think of creating a ESLint rule for this?

Reply:
🤔  I don't think it should be an error, a warning if anything.